ssld/ssld.c

```diff
@@ -112,40 +159,36 @@ typedef struct _conn {
     unsigned long long plain_in;
     unsigned long long plain_out;
     uint8_t flags;
-    void *stream;
 } conn_t;
 
-#define FLAG_SSL	0x01
-#define FLAG_ZIP	0x02
-#define FLAG_CORK	0x04
-#define FLAG_DEAD	0x08
-#define FLAG_SSL_W_WANTS_R 0x10	/* output needs to wait until input possible */
-#define FLAG_SSL_R_WANTS_W 0x20	/* input needs to wait until output possible */
-#define FLAG_ZIPSSL	0x40
+#define FLAG_SSL           0x01
+//define FLAG_ZIP          0x02 /* Removed */
```

Comment:
left it in to explain the gap between ssl and cork, and didn't want to change all the flags to avoid the gap